### PR TITLE
Improved Docstrings

### PR DESCRIPTION
**Pull request by rsdel2007**
The aim of this PR is to improve docstrings by fixing some spelling errors.

### CODE REVIEW

**Comment by ericmjl** [comment]
All checks look good to me. Are you ready for this to be merged, @rsdel2007? Please let me know, and I'll hit the big green button.

**Comment by rsdel2007** [comment]
@ericmjl, I have checked the others. Please go ahead.

File: pymc3/variational/updates.py [hunk 1]
```diff
@@ -234,7 +234,7 @@ def apply_momentum(updates, params=None, momentum=0.9):
     updates : OrderedDict
         A dictionary mapping parameters to update expressions
     params : iterable of shared variables, optional
-        The variables to apply momentum to. If omitted, will apply
+        The variables to apply momentum too. If omitted, will apply
```
Comment (eigenfoo) [inline pymc3/variational/updates.py]: Wasn't this originally correct? Shouldn't it be "the variables to apply momentum to"?

Comment (ericmjl) [reply pymc3/variational/updates.py]: Gooooooood catch, @eigenfoo! I was not looking closely enough there.

Comment (ericmjl) [reply pymc3/variational/updates.py]: I have directly edited the PR. This is fixed now.

File: pymc3/variational/updates.py [hunk 2]
```diff
@@ -344,7 +344,7 @@ def apply_nesterov_momentum(updates, params=None, momentum=0.9):
...
```
Comment (eigenfoo) [inline pymc3/variational/updates.py]: Also here 😄

```suggestion
        The variables to apply momentum to. If omitted, will apply
```

Comment (ericmjl) [reply pymc3/variational/updates.py]: You are a meticulous person, @eigenfoo :smile:!

**Review by eigenfoo — APPROVED** [review]
Looks great! I'll merge this once tests pass. Thanks @ericmjl, and congrats @rsdel2007 on your first contribution to PyMC3! 😄

**Comment by ericmjl** [comment]
Awww, but but, I wanted to hit that big green button!! :smile_cat:

**Comment by eigenfoo** [comment]
Hahah, I'll let you do it next time! 😉

**Pull request merged**